bilby/gw/likelihood.py

```diff
@@ -1434,17 +1434,22 @@ class RelativeBinningGravitationalWaveTransient(GravitationalWaveTransient):
     """
 
     def __init__(self, interferometers, waveform_generator,
-                 initial_parameters={}, parameter_bounds={}, chi=1,
-                 epsilon=.5):
+                 initial_parameters={}, parameter_bounds={},
+                 chi=1, epsilon=.5, priors=None,
+                 distance_marginalization=False,
+                 phase_marginalization=False,
+                 time_marginalization=False,
+                 jitter_time=False):
```

Comment:
For this portion, I think it would be better to make it similar to #26. It would probably be best if you merge those changes in to this. I can do this if you aren't confident enough

Reply:
The jitter issue wasn't resolved, right?
Even after the fix, the same error was popping up.